codemarathon-2-dev-firebase-adminsdk.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Add it to .gitignore please and recommit using `--amend`, then `push -f`. Also remove this file from git using `git rm --cached`.